source/server/server.cc

```diff
@@ -273,25 +273,24 @@ void InstanceBase::updateServerStats() {
   server_stats_->total_connections_.set(listener_manager_->numConnections() +
                                         parent_stats.parent_connections_);
   server_stats_->days_until_first_cert_expiring_.set(
-      sslContextManager().daysUntilFirstCertExpires().value_or(0));
+      (*ssl_context_manager_).daysUntilFirstCertExpires().value_or(0));
```

Comment:
I don't see why this change needs to be a thing, the original doesn't seem like something clangtidy would object to. (Same confusion throughout this file, just seems like it's all risking creating future-work by removing a level of convenience abstraction.)

Reply:
yeah - i think this was over eager bot - will revert this